golem_messages/message.py

```diff
@@ -883,17 +883,15 @@ class SubtaskResultAccepted(Message):
 
     __slots__ = [
         'subtask_id',
-        'reward'
+        'payment_ts'
     ] + Message.__slots__
 
-    def __init__(self, subtask_id=0, reward=0, **kwargs):
+    def __init__(self, subtask_id: str="", payment_ts: int=0, **kwargs):

```

Comment:
Since the signature changes Is it possible to drop this `__init__` as in `ForceReportComputedTask`?

Reply:
Discussed offline, decided to go with #61